graylog2-server/src/main/java/org/graylog/integrations/aws/service/KinesisService.java

```diff
@@ -157,7 +157,7 @@ public KinesisHealthCheckResponse healthCheck(KinesisRequest request) throws Exe
         final List<Record> records = retrieveRecords(request.streamName(), kinesisClient);
 
         if (records.isEmpty()) {
-            LOG.warn("The Kinesis stream [{}] is empty. Proceeding with setup at the user's own risk.", request.streamName());
+            LOG.debug("The Kinesis stream [{}] is empty.", request.streamName());
```

Comment:
Bumped this one down to debug, since using an empty stream is a common non-error case.